### ISSUE TITLE
Align name of an unlinked token and its synthetic actor

### ISSUE THREAD

**Issue by admiralnlson** [foundryvtt/foundryvtt#11544]
### User Experience

As of 12.330, for unlinked tokens, the name of the token (`myToken.name`) and the name of its synthetic actor (`myToken.actor.name`) are not kept aligned.

I would argue they should always be one and the same.

My proposal is to propagate changes to the name of an unlinked token to its synthetic actor, and vice versa.
And to take similar measures if necessary when the token is first created / cloned, or when a token changes from linked to unlinked.

See also conversation about this in `#v12-feedback`: https://discord.com/channels/170995199584108546/1184176344276406292/1270250168394059878

**Comment by aaclayton** [comment]
Certainly we would not want to force the token name and base actor name into alignment, but I do see an argument for making it easier to update the token name and the name of the unlinked actor delta. Worth considering.

**Comment by admiralnlson** [comment]
@BoltsJ 
Me too! :)
Please note I am talking here about the _actor delta_ (a.k.a. _synthetic actor_) which is specific to and contained within each unlinked token (and has uuid like `Scene.sMb4JNoxFt76NGRY.Token.o8TIs9O5QSLmjAw1.Actor.pxGcQdukH5gp2waX`), not the _base actor_ visible in the Actors navbar (which has a uuid like `Actor.pxGcQdukH5gp2waX`).
As mentioned by Atropos above, it would make no sense to update the base actor when an unlinked token name changes.

**Comment by BoltsJ** [comment]
I understand what the feature request is, and it's a behavior that I don't want, at least not my default, even for unlinked tokens. Changing the name on an unlinked actor would be really annoying for my use cases.

**Comment by admiralnlson** [comment]
After the exchange with @BoltsJ here and in Discord (link: https://discord.com/channels/170995199584108546/1184176344276406292/1270410404140679188 )
I've changed my mind on
> I would argue they should always be one and the same.

The canvas is real estate sometimes. Having the option to name a token "Steve" and have a longer name e.g. "Steve Tumblebreeze, Merchant" as actor name displayed in the sheet makes a lot sense.

So I now definitely agree it would have to be an option, if anything is implemented in Core at all.

**Comment by aaclayton** [comment]
I agree that when an unlinked token is initially created the `token.delta.name` should be initialized to `token.name`. They could diverge after that point, but as far as creation of the initial delta is concerned I think they should match. 

This is especially true when the token was created using a prepended adjective or appended incrementing number.
